### ISSUE TITLE
--fts list-of-columns option

### ISSUE THREAD

**Issue by simonw** [simonw/csvs-to-sqlite#7]
Creates a FTS5 table populated with the content of the specified columns: https://sqlite.org/fts5.html (falls back on FTS4 if FTS5 is not supported)

Would make it trivial to add fulltext search to data from a CSV file.

**Comment by simonw** [comment]
Should generate these virtual tables too (maybe just thev"cop") variant - provided they don't inflate the file size: https://sqlite.org/fts5.html#the_fts5vocab_virtual_table_module

**Comment by simonw** [comment]
`--fts` will use the best available FTS implementation. `--fts4` will use FTS4, so even if you are on an FTS5 installation you can create a file that is backwards-compatible with older Pythons.

Looks like the only way to detect these extensions is to open a `:memory:` database and run commands on it and see if they throw errors: https://stackoverflow.com/questions/2482868/detect-the-fts3-extension-in-sqlite3

**Comment by simonw** [comment]
```
$ sqlite3 :memory:
SQLite version 3.8.10.2 2015-05-20 18:17:19
Enter ".help" for usage hints.
sqlite> CREATE VIRTUAL TABLE v USING FTS3 (t TEXT);
sqlite> CREATE VIRTUAL TABLE v2 USING FTS4 (t TEXT);
sqlite> CREATE VIRTUAL TABLE v3 USING FTS5 (t TEXT);
Error: no such module: FTS5
```

**Comment by simonw** [comment]
Implemented in f728625354b80a4724631cbcda5fcff573bb8f7f
